service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/EpisodeOfCareMapperTest.java

```diff
@@ -101,10 +101,10 @@ public void mapEpisodeOfCareNoResponsibleParty() {
 
     public Optional<EpisodeOfCare> mapEpisodeOfCare(POCDMT000002UK01ClinicalDocument1 clinicalDocument, Reference subject) {
         POCDMT000002UK01EncompassingEncounter encompassingEncounter = clinicalDocument.getComponentOf()
-            .getEncompassingEncounter();
+                .getEncompassingEncounter();
```

Comment:
please remove additional spacing here

Reply:
This is the code formatter, not added intentionally